### PR TITLE
lxappearance: default to Gtk3

### PR DESCRIPTION
**Pull request by romildo**
###### Motivation for this change

- make Gtk3 the default
- remove `lxappearance-gtk3`
- add `lxappearance-gtk2`
- format with `nixpkgs-fmt`
- use pkg-config instead of the alias `pkgconfig`
- add romildo as maintainer

See also https://github.com/NixOS/nixpkgs/issues/26909#issuecomment-459002407

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by pbogdan** [comment]
@romildo Do you recall why we need to keep both variants and not just drop the gtk2 one?

**Comment by romildo** [comment]
If I am not wrong at the time the Gtk2 were still very used.

It seems that the Gtk3 version does not list themes which do not target Gtk3, like the Raleigh theme.

But I would have no problem in dropping the Gtk2 version.

File: pkgs/desktops/lxde/core/lxappearance/default.nix [hunk 1]
```diff
@@ -1,26 +1,42 @@
...
-    description = "A lightweight program for configuring the theme and fonts of gtk applications";
+  meta = with stdenv.lib; {
+    description = "Lightweight program for configuring the theme and fonts of gtk applications";
     homepage = "https://lxde.org/";
-    maintainers = [ stdenv.lib.maintainers.hinton ];
-    platforms = stdenv.lib.platforms.linux;
-    license = stdenv.lib.licenses.gpl2;
+    license = licenses.gpl2;
+    platforms = platforms.linux;
+    maintainers = with  maintainers; [ hinton romildo ];
```
Comment (NickHu) [inline pkgs/desktops/lxde/core/lxappearance/default.nix:40]: Does `nixpkgs-fmt` really insert this double space here?

Comment (romildo) [reply pkgs/desktops/lxde/core/lxappearance/default.nix]: Fixed. There were some minor manual editions that was not mentioned in the commit message.

**Pull request merged**